src/test/java/htsjdk/samtools/util/GZIIndexTest.java

```diff
@@ -72,16 +72,18 @@ public void testWriteIndex(final File indexFile, final int exprectedBlocks) thro
         return new Object[][] {
                 // TODO - requires more test files
                 {new File("src/test/resources/htsjdk/samtools/block_compressed.sam.gz"),
-                        new File("src/test/resources/htsjdk/samtools/block_compressed.sam.gz.gzi")}
+                        new File("src/test/resources/htsjdk/samtools/block_compressed.sam.gz.gzi")},
+                {new File("src/test/resources/htsjdk/samtools/reference/Homo_sapiens_assembly18.trimmed.fasta.gz"),
+                        new File("src/test/resources/htsjdk/samtools/reference/Homo_sapiens_assembly18.trimmed.fasta.gz.gzi")}
         };
     }
 
     @Test(dataProvider = "filesWithIndex")
     public void testCreateIndex(final File fileToIndex, final File expectedIndex) throws Exception {
```

Comment:
this test is now misnamed, and create index isn't tested

Reply:
Renamed test.